speakeasy/binemu.py

```diff
@@ -781,6 +781,13 @@ def write_mem_string(self, string, address, width=1):
         enc_str = string.encode(encode)
         self.mem_write(address, enc_str)
 
+    def read_ptr(self, address):
+        val = self.mem_read(address, self.ptr_size)
+        return int.from_bytes(val, 'little')
+
+    def write_ptr(self, address, val):
+        self.mem_write(address, val.to_bytes(self.ptr_size, 'little'))
+
```

Comment:
i was surprised these didn't already exist. did i miss them somewhere? is there a better way to do this?

Reply:
No, seems like that was an oversight. This is very useful.